src/market/market.module.ts

```diff
@@ -26,11 +32,26 @@ import { WorkloadDemandDirectorConfigOptions } from "./demand/options";
 import { BasicDemandDirectorConfig } from "./demand/directors/basic-demand-director-config";
 import { PaymentDemandDirectorConfig } from "./demand/directors/payment-demand-director-config";
 import { GolemUserError } from "../shared/error/golem-error";
-import { createAbortSignalFromTimeout } from "../shared/utils/abortSignal";
 import { MarketOrderSpec } from "../golem-network";
-import { NetworkModule, INetworkApi } from "../network";
+import { INetworkApi, NetworkModule } from "../network";
+
+export interface IOfferProposalEvents {
```

Comment:
This seems to be redundant as it is covered with IMarketEvents - should be dropped from the codebase.